pan_firewall/assets/logs/pan.firewall.yaml

```diff
@@ -230,4 +385,1028 @@ pipeline:
+          name: Activity ID Category
+          enabled: true
+          categories:
+            - filter:
+                query: "@subtype:start"
+              name: "1"
+            - filter:
+                query: "@subtype:end"
+              name: "2"
+            - filter:
+                query: "@subtype:drop"
+              name: "3"
+            - filter:
+                query: "@subtype:deny"
+              name: "5"
+          target: ocsf.activity_id
+        - type: category-processor
+          name: map activity name
+          enabled: true
+          categories:
+            - filter:
+                query: "@subtype:start"
+              name: "Open"
+            - filter:
+                query: "@subtype:end"
+              name: "Close"
+            - filter: "@subtype:drop"
+              name: "Reset"
+            - filter: "@subtype:deny"
+              name: "Refuse"
+          target: ocsf.activity_name
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.destination.ip' to 'ocsf.dst_endpoint.ip'
+          enabled: true
+          sources:
+            - network.destination.ip
+          sourceType: attribute
+          target: ocsf.dst_endpoint.ip
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.client.ip' to 'ocsf.src_endpoint.ip'
+          enabled: true
+          sources:
+            - network.client.ip
+          sourceType: attribute
+          target: ocsf.src_endpoint.ip
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.destination.port' to 'ocsf.dst_endpoint.port'
+          enabled: true
+          sources:
+            - network.destination.port
+          sourceType: attribute
+          target: ocsf.dst_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: remap 'network.client.port' to 'ocsf.src_endpoint.port'
+          enabled: true
+          sources:
+            - network.client.port
+          sourceType: attribute
+          target: ocsf.src_endpoint.port
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: string-builder-processor
+          name: 99 - in attribute ocsf.connection_info.direction_id
+          enabled: true
+          template: "99"
+          target: ocsf.connection_info.direction_id
+          replaceMissing: false
+        - type: string-builder-processor
+          name: 0 - in attribute ocsf.connection_info.direction_id
+          enabled: true
+          template: "0"
+          target: ocsf.connection_info.direction_id
```

Comment:
Duplicate string-builder targeting the same attribute. I'd probably set it as `0` and then `ocsf.connection_info.direction` as `Unknown`.